tests/phpunit/integration/Modules/Analytics/SettingsTest.php

```diff
@@ -300,6 +300,32 @@ function () use ( &$filter_return ) {
 		$this->assertTrue( $settings->get()['canUseSnippet'] );
 	}
 
+	public function test_tracking_disabled() {
+		$context  = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
+		$settings = new Settings( new Options( $context ) );
+		$settings->register();
+		$analytics = new Analytics( $context );
```

Comment:
Looks like this variable is not needed, is it? Let's get rid of it if we don't use it.

Reply:
Apologies for missing that out, sorted now. Thank you for the review @eugene-manuilov!